lib/crypto/packaged.py

```diff
@@ -51,6 +52,17 @@ def get_endpoint(server):
     return u'{server}/1.0/sign_addon'.format(server=server)
 
 
+def get_id(addon):
+    """Return the addon GUID if <= 64 chars, or its sha256 hash otherwise.
+
+    We don't want GUIDs longer than 64 chars: bug 1203365.
+    """
+    guid = addon.guid
+    if len(guid) <= 64:
+        return guid
+    return hashlib.sha256(guid).hexdigest()
```

Comment:
Does the id need to be prefixed with sha256: or something so that the client can detect if it has been hashed? Or will it be able to do that by looking at other data and know this has been hashed?